Sources/Security/VerificationReason.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+
+public enum VerificationReason {
+    case invalidPublicKey(String)
+    case signatureRequestedButNotProvided(String)
+    case signatureNotBase64(String)
+    case signatureInvalidSize(String)
+    case signatureFailedVerification
+    case intermediateKeyFailedVerification(String)
+    case intermediateKeyFailedCreation(String)
+    case intermediateKeyExpired(String, String)
+    case intermediateKeyInvalid(String)
+    case intermediateKeyCreating(String, String)
+    case requestDateMissingFromHeaders(String)
+}
+
+// Clase wrapper para Objective-C
```

Comment:
I guess, I am learning Spanish now 😜

Ha-ha no worries, I know the basics already from the Dualingo lessons